content/get.md

```diff
@@ -13,51 +13,50 @@ Currently (June 2021), we support machine configurations for:
 -   saga (sigma2, Norway)
 -   fram (sigma2, Norway)
 
-If your machine is not in the list and you would like us to support it,
+If your machine is not on the list and you would like us to support it,
 please contact us.
 
 ## How to get CTSM (for users)
 A user is a person that runs CTSM without modifying the source code. To get the FATES EMERALD platform version, CLONE from NordicESM hub
 
     [~/HOME]$ git clone -b release-clm5.0 https://github.com/NordicESMhub/ctsm.git ${HOME}/ctsm_fates_emerald
 
-In this example we are checking out the release-clm5.0 tag and create a
+In this example, we are checking out the release-clm5.0 tag and create a
 new local branch (recommended). The destination of the checkout is a
 directory (e.g. ctsm\_fates\_emerald) in our home directory.
 
 ### How to get a specific branch
 Change into the created ctsm directory
 
-    cd ${HOME}/ctsm_fates_emerald
+    [~/HOME]$ cd ctsm_fates_emerald
 
 Check all existing branches
 
-    git branch --all
+    [~/HOME/ctsm_fates_emerald]$ git branch --all
 
-To checkout the FATES EMERALD platform (in this example release 2.0.1)
+To check out the FATES EMERALD platform (in this example release 2.0.1)
 into a new local branch (e.g. new\_branch\_name)
 
-    git checkout release-emerald-platform2.0.1 -b new_branch_name
+    [~/HOME/ctsm_fates_emerald]$ git checkout release-emerald-platform2.0.1 -b new_branch_name
 
 For later reference, it is useful to choose new\_branch\_name according
 to function and include the version and your username, e.g.
 username\_release-emerald-platform2.0.1.
 
 To fetch the proper externals (CIME, FATES, etc.) run
```

Comment:
Also here, a link to some background information would be nice, IMO

Reply:
I will ask you to take a look again and check if we have sufficient `git`-related stuff when the total webpage is done.